test/support/flow_test_helper.rb

```diff
@@ -154,12 +154,16 @@ def outcome_title
       outcome_presenter.title
     end
 
+    def outcome_next_steps
+      outcome_presenter.next_steps
+    end
+
     def outcome_body_text
       Nokogiri::HTML::DocumentFragment.parse(outcome_body).text
     end
 
     def outcome_text
-      [outcome_title, outcome_body_text].reject(&:blank?).join("\n")
+      [outcome_title, outcome_body_text, outcome_next_steps].reject(&:blank?).join("\n")
```

Comment:
As `outcome_next_steps` is HTML could we have this as a `outcome_next_steps_text` method that strips the HTML for consistency?

(Just a quick drive-by comment as I know Rosa is doing a more thorough review)

Reply:
Oops! Missed this. I think was using `title` as a template. Will update and fix up.